### PR TITLE
ZER-96/Add footer on admin side

### PR DESCRIPTION
**Pull request by anko20094**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/browse/ZER-96)

## Code reviewers

- [ ] @olegsobchuk 

### Second Level Review

- [ ] @github_username

## Summary of issue

Add Footer to admin side

As admin I should see footer clued to the bottom of page

With height 100px  and some text within this block

## Summary of change

Add fixed on bottom footer to admin side

## CHECK LIST
- [x]  СI passed
- [x]  Сode coverage >=95%
- [x]  PR is reviewed manually again (to make sure you have 100% ready code)
- [x]  All reviewers agreed to merge the PR
- [x]  I've checked new feature as logged in and logged out user if needed
- [x]  PR meets all conventions

### CODE REVIEW

File: app/views/layouts/admin.html.slim [hunk 1]
```diff
@@ -8,7 +8,9 @@ html
     = csp_meta_tag
     = stylesheet_link_tag 'application', media: 'all', 'data-turbolinks-track': 'reload'
     = stylesheet_pack_tag 'application', media: 'all', 'data-turbolinks-track': 'reload'
+    
     = javascript_pack_tag 'application', 'data-turbolinks-track': 'reload'
   body.admin
     .container.admin-container
-      = yield
+    = yield
+  = render 'layouts/admin_footer'
```
Comment (olegsobchuk) [inline app/views/layouts/admin.html.slim:16]: add empty line to the end of this file

File: .gitignore [hunk 2]
```diff
@@ -54,3 +54,6 @@ yarn-debug.log*
 /config/credentials/staging.key
 
 /config/credentials/uat.key
+
+# Ignore temporary file from redis 'dump.rdb'
+dump.rdb
```
Comment (olegsobchuk) [inline .gitignore:59]: why do you need this? It's something wrong with your Redis settings

**Review by AnastasiiaBilynska — DISMISSED** [review]

**Review by olegsobchuk — APPROVED** [review]
LGTM

**Review by AnastasiiaBilynska — APPROVED** [review]

**Review by olegsobchuk — APPROVED** [review]
LGTM

**Pull request merged**